contracts/vault/StrategySplitterV2.sol

```diff
@@ -385,9 +389,7 @@ contract StrategySplitterV2 is ControllableV3, ReentrancyGuard, ISplitter {
       emit Loss(lowStrategy, strategyLossOnWithdraw);
     }
 
-    (address topStrategy, uint strategyLossOnInvest, uint strategyBalanceAfterInvest) = _investToTopStrategy(
-      false // we assume here, that total-assets-amount of the strategy was just updated in withdraw above
-    );
+    (address topStrategy, uint strategyLossOnInvest, uint strategyBalanceAfterInvest) = _investToTopStrategy(true);
```

Comment:
true means that invested assets amount will be recalculate
if lowStrategyBalance != 0 then withdraw-call already recalculated it
probably  we can write the code a bit more efficiently: 
uint x = (lowStrategyBalance != 0);
if (x) { ... withdraw ... }
_investToTopStrategy(!x);